main/src/cgeo/geocaching/connector/gc/GCWebAPI.java

```diff
@@ -389,7 +443,13 @@ static MapSearchResultSet searchMap(@NonNull final Viewport viewport) {
                     put("logText", logInfo).
                     put("usedFavoritePoint", formatBoolean(addToFavorites));
 
-            final PostLogResponse response = postAPI("/web/v1/geocache/" + StringUtils.lowerCase(geocache.getGeocode()) + "/GeocacheLog", params, PostLogResponse.class).blockingGet();
+            GeocacheLog geocacheLog = new GeocacheLog(geocache.getCacheId(), geocache.getGeocode(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'geocacheLog' should be declared final.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=5262679)